### PR TITLE
Added check-suite checking

### PR DESCRIPTION
**Pull request by RDIL**
Check if the suite passed or not

### CODE REVIEW

**Comment by RDIL** [comment]
Wait no - we might need it to check if the check-suite has passed.

**Comment by RDIL** [comment]
Now it checks to see if the check-suite is passing or not

**Comment by fkorotkov** [comment]
Don't we want to report failures? Also let's move all the logic to `hooks.sh`.

**Comment by RDIL** [comment]
Yes, good idea.  My thoughts is we use the other action to check if the suite is passing or not, and if it is, run `bash ./hooks.sh did-not-fail`

**Comment by RDIL** [comment]
Alright I think I fixed it all up, should now be ready for review.

File: hooks.sh [hunk 1]
```diff
@@ -6,7 +6,17 @@ git config --global user.email "action@github.com"
 git config --global user.name "GitHub Action"
 git remote set-url origin https://x-access-token:$GITHUB_TOKEN@github.com/$GITHUB_REPOSITORY.git
 
-EMBED_COLOR=3066993
+# check if it passed or not
+$element = jq -r ".check_suite.app.name" "$GITHUB_EVENT_PATH" | grep -Eq "^$APP_NAME$"
```
Comment (fkorotkov) [inline hooks.sh]: where is `$element` used?

Comment (RDIL) [reply hooks.sh]: It isn't, just not sure where to put it since I took it from the other actions' dockerfile.  I'll remove it if it is no longer used.

**Comment by fkorotkov** [comment]
What are you trying to achieve here? It's not clear from the PR title or description in the first comment.

**Comment by RDIL** [comment]
I'm trying to make it so it can check if the suite passed or not
sorry, just updated the description

**Review by fkorotkov — CHANGES_REQUESTED** [review]
Let's remove `$element` and then squash merge it.

**Comment by RDIL** [comment]
Removed it and squashed.

**Pull request merged**